### DISCUSSION TITLE
Kiosk PXE images with persistent home directories

### DISCUSSION THREAD

**Discussion by ITaluone** [ltsp/ltsp#776] [Q&A] [answered]
Hi 

Trying for quite a long time now to set up a pxe image with user01 to userXX with persistent home directories, and I am stuck..

What I want to achieve is:
* autologin user userXX
* store all user home dirs inside one dir on the pxe server (like below /nfs/homes/[user01...userXX])
* no need to add every user on the server

Is this possible? I have tried several tutorials regarding nfs from here: https://ltsp.org/man/ltsp-nfs/#examples

And if it is possible: How do I mount this home dirs on bootup? Now the boot process is stuck at "mounting /home" and times out after 1:30min.

Thanks

**Comment by alkisg** [comment, marked as answer]
Hi @ITaluone, search the discussions here, it's been discussed many times.
You need **one** user on the server, and of course multiple home directories, mounted in ltsp.conf like this:

```ini
[mac-of-pc01]
FSTAB_HOME="server:/home/user/pc01 /home nfs defaults,nolock 0 0"

[mac-of-pc02]
FSTAB_HOME="server:/home/user/pc02 /home nfs defaults,nolock 0 0"
```

And on the server, the following dirs must exist and be owned by user:

```
/home/user/pc01/user
/home/user/pc02/user
/home/user/pc03/user
```

**Reply by ITaluone** [reply]
Thank you :)
